usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -112,6 +111,22 @@ def last_closed_monthly_submission_dates(self):
     def last_closed_quarterly_submission_dates(self):
         return get_last_closed_submission_date(is_quarter=True)
 
+    @cached_property
+    def final_submissions_query_filters(self):
+        return filter_by_latest_submissions()
+
+    @cached_property
+    def all_closed_defc_submissions(self):
```

Comment:
Future readers would appreciate a comment here ... suggesting that 
> this filters should only be used when looking at submission data that includes DEF Codes, which only started appearing in submission for FY2020 P07 (Apr 1, 2020) and after

Reply:
👍  Adding docstring